apps/src/templates/VersionRow.jsx

```diff
@@ -21,58 +25,75 @@ export default class VersionRow extends React.Component {
     return timestamp.toString();
   }
 
+  getUrlAttributes() {
+    const user_id = queryParams('user_id');
+    const viewAs = queryParams('viewAs');
+    return (
+      (this.props.isLatest ? '' : '?version=' + this.props.versionId) +
```

Comment:
I missed this the first time, but this will be in an invalid format if `isLatest` is true and `user_id` or `viewAs` are present (example result `&user_id=2` - you'll be missing the initial ?). You could use the `query-string` library and call stringify on the object https://github.com/sindresorhus/query-string#stringifyobject-options
```
queryString.stringify({foo: [1, 2, 3]});
//=> 'foo=1&foo=2&foo=3'
```

Reply:
I don't think this state is possible, but you're right that this is more complete/future proof design. I'll update!

Reply:
Updated!